inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/SourceRequest.java

```diff
@@ -72,8 +72,8 @@ public class SourceRequest {
     @ApiModelProperty("Inlong cluster name")
     private String inlongClusterName;
 
-    @ApiModelProperty("Inlong cluster node tag")
-    private String inlongClusterNodeTag;
+    @ApiModelProperty("Inlong cluster node label")
```

Comment:
Add explanation above. like @ApiModelProperty(value = "MQ resource", notes = "in inlong group, TubeMQ corresponds to Topic, Pulsar corresponds to Namespace").